### ISSUE TITLE
mingw64 neon-git

### ISSUE THREAD

**Issue by slyfox1186** [m-ab-s/media-autobuild_suite#2028]
``` bash
05:25:56 └ ffmpeg-git  ........................................ [Updated]
05:26:00   Running git clone for neon...
05:26:04 ┌ neon git  ................................. [Recently updated]
05:26:04 ├ Running autogen...
05:26:16 ├ Running configure...
05:27:17 ├ Running make...
Likely error (tail of the failed operation logfile):
../../src/ne_socket.c:2074:44: note: each undeclared identifier is reported only once for each function it appears in
../../src/ne_socket.c:2075:31: error: 'SHUT_WR' undeclared (first use in this function)
 2075 |      (flags == NE_SOCK_SEND ? SHUT_WR : SHUT_RDWR));
      |                               ^~~~~~~
../../src/ne_socket.c:2075:41: error: 'SHUT_RDWR' undeclared (first use in this function)
 2075 |      (flags == NE_SOCK_SEND ? SHUT_WR : SHUT_RDWR));
      |                                         ^~~~~~~~~
make[1]: *** [Makefile:69: ne_socket.lo] Error 1
make[1]: Leaving directory '/build/neon-git/build-64bit/src'
make: *** [Makefile:88: subdirs] Error 2
make failed. Check E:/MAB-E/build/neon-git/build-64bit/ab-suite.make.log
This is required for other packages, so this script will exit.
05:27:21   Creating diagnostics file...

All relevant logs have been anonymously uploaded to https://0x0.st/-ytW.zip
Copy and paste [logs.zip](https://0x0.st/-ytW.zip) in the GitHub issue.
Make sure the suite is up-to-date before reporting an issue. It might've been fixed already.
Try running the build again at a later time.
```

[logs.zip](https://github.com/jb-alvarado/media-autobuild_suite)

**Comment by slyfox1186** [comment]
My apologies I see this is already posted.

**Comment by Biswa96** [comment]
Duplicate of https://github.com/m-ab-s/media-autobuild_suite/issues/2026
